pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/data/Playlist.kt

```diff
@@ -863,6 +858,17 @@ data class Playlist(val title: String, val items: List<DemoItem>, val languageTa
             languageTag = "en-CH",
         )
 
+        val DashIfStream = Playlist(
+            title = "DashIf stream",
```

Comment:
```suggestion
        val dashIfStream = Playlist(
            title = "DashIf streams",
```

Reply:
`DashIfStream` is correct as all other Playlist begin with a capital letter.